app/controllers/items/detail.js

```diff
@@ -32,6 +32,16 @@ export default GoodcityController.extend(singletonItemDispatchToGcOrder, {
     }
   ),
 
+  allowPublish: Ember.computed(
+    "model.isSingletonItem",
```

Comment:
`Ember.computed('model.{isSingletonItem, availableQty}')`

Reply:
oh that's cool :)

Reply:
I'll merge as is, I'll tweak in my upcoming batch of changes 👍